doc/charm++/manual.rst

```diff
@@ -2701,6 +2701,74 @@ applications. RotateLB and RandCentLB are more useful for debugging
 object migrations. The compiler and runtime options are described in
 section :numref:`lbOption`.
 
+**TreeLB and its configuration**
+
+Recent changes in the LB infrastructure add support for a user-configurable
+hierarchical load balancing using TreeLB. While the legacy centralized
+strategies above are still supported, TreeLB allows performing load
+balancing at multiple levels and supports load balancing trees with different
+levels. For example, a 2-level tree consists of PEs and a root while a
+3-level tree consists of PEs, processes and a root at the top. A 4-level
+tree consists of PEs, processes, ProcessGroup and a root. The load balancing
+strategy to be used at each level and frequency at which to invoke LB
+at each level can be specified using command line parameters or using a json
+config file. We provide examples of these below:
```

Comment:
There are only examples of the json files, not the command line parameters?

Reply:
I've removed the text on command line parameters since that is not supported currently.